src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceVersion2.java

```diff
@@ -70,6 +71,18 @@ public DataAccessRequestResourceVersion2(
     this.matchService = matchService;
   }
 
+  @GET
+  @Produces("application/json")
+  @RolesAllowed(ADMIN)
```

Comment:
In the old resource class, this was a Permit All. Why restrict it to admins here?

Reply:
I'm not sure I think maybe I thought it was permit all by mistake, I'll change it back